### ISSUE TITLE
Unable to override text-align property for .MathJax_Display class.

### ISSUE THREAD

**Issue by wllmtrng** [mathjax/MathJax#1261]
The following code:

```
$$x = {-b \pm \sqrt{b^2-4ac} \over 2a}.$$
```

generates the html output:

```html
<div class="MathJax_Display" style="text-align: center;">....</div>
```

The definition of the style="text-align: center;" forces the developer to use **!important** when overriding the text-align property for the MathJax_Display class. This is bad practice in my opinion and leads to confusion.

**Comment by pkra** [comment]
tl;dr: there's a configuration option to control alignment of display-style equations -- set `displayAlign` in the core http://docs.mathjax.org/en/latest/options/hub.html

Isolation from surrounding CSS is necessary to realize complex math layout reliably -- it's delicate work and surrounding CSS can screw it up quickly. The price is that developers have to declare CSS in a way that MathJax can integrate during layout, see http://docs.mathjax.org/en/latest/CSS-styles.html.  Using TeX input (which does not fit into HTML and CSS at all) does not make things easier for a developer; MathML (which TeX gets converted into anyway) is usually easier for that kind of thing.

**Comment by wllmtrng** [comment]
My apologies, the problem is due to the parent app not reading the custom reconfiguration of the core settings correctly.

Recommend closing.

**Comment by pkra** [comment]
Thanks for following up. Glad you could fix the problem.
